src/Framework/FieldsAPI/Section.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Give\Framework\FieldsAPI;
+
+/**
+ * @unreleased
+ */
+class Section extends Group
+{
+    /**
+     * @unreleased
+     */
+    const TYPE = 'section';
+
+    /**
+     * @unreleased
+     *
+     * @param  Field  ...$fields
+     * @return $this
+     */
+    public function addFields(Field ...$fields): Section
```

Comment:
Probably should just be a node as it can be a `Field`, `Element`, or `Group`. I think the `Group` parent class already has a method that appends nodes.

Reply:
Yes `Group` already has `append`.  I thought we talked about making it more specific with `Section` like I was doing here but If that's not the case I can just remove this and use append.

Reply:
I don't really see the value right now. Let's add something like this later if a use-case comes up.

Reply:
Okay, just reverted 6eacbd97